single.just

```diff
@@ -46,7 +46,7 @@ sign hdPath='0':
   echo ""
 
   forge build
-  $(git rev-parse --show-toplevel)/bin/eip712sign --ledger --hd-paths "m/44'/60'/{{hdPath}}'/0/0" -- \
+  $(go env GOPATH)/bin/eip712sign --ledger --hd-paths "m/44'/60'/{{hdPath}}'/0/0" -- \
```

Comment:
This is contradictory to the README file in the root directory: https://github.com/ethereum-optimism/superchain-ops?tab=readme-ov-file#installing-eip712sign

Which specifically instructs us to install it in the go directory, and lots of our teammates are having the issue when executing the just task

Reply:
Also it seems like I don't have permission to reopen the PR @zchn

Reply:
Good point @0x00101010. I think we will probably want to open a PR to remove the eip712 installation step, since we'll have the local eip712sign installation as part of `just install`—can you confirm @zchn?